### PR TITLE
Add node group roles

### PR DESCRIPTION
**Pull request by Thiblef**
Functional tests in progress

### CODE REVIEW

File: Backoffice/Security/Authorization/Voter/StatusableVoter.php [hunk 1]
```diff
@@ -21,7 +21,9 @@ class StatusableVoter implements VoterInterface
      */
     public function supportsAttribute($attribute)
     {
-        return 0 === strpos($attribute, 'ROLE_');
+        return 0 === strpos($attribute, 'ROLE_')
+        && FALSE === strpos($attribute, 'ROLE_ACCESS_TREE_NODE')
```
Comment (nicolasThal) [inline Backoffice/Security/Authorization/Voter/StatusableVoter.php]: could you put false in lower case

File: Backoffice/Security/Authorization/Voter/StatusableVoter.php [hunk 2]
```diff
@@ -21,7 +21,9 @@ class StatusableVoter implements VoterInterface
...
+        && FALSE === strpos($attribute, 'ROLE_ACCESS_CREATE_NODE');
```
Comment (nicolasThal) [inline Backoffice/Security/Authorization/Voter/StatusableVoter.php]: Does the node suppression has the same restriction on the node status ?

Comment (Thiblef) [inline Backoffice/Security/Authorization/Voter/StatusableVoter.php]: No it doesn't

**Comment by Thiblef** [comment]
New PR created to clean commits :  https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1254

**Pull request closed without merging**